packages/jest/migrations.json

```diff
@@ -136,6 +136,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "14.5.3": {
+      "version": "14.5.3-beta.0",
+      "packages": {
+        "jest": {
```

Comment:
I guess this should be `ts-node`, isn't it?

Reply:
yes, this should be ts-node. Thanks for catching!